### PR TITLE
Return messages as json objects

### PR DESCRIPTION
**Pull request by zjayee**
# Description

Change websocket to return a json object instead of the plain message

## Type of change
- New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
- manual testing

### CODE REVIEW

File: src/main/java/callhub/connect/drivers/MessageWebSocketAccess.java [hunk 1]
```diff
@@ -1,24 +1,43 @@
 package callhub.connect.drivers;
+import com.google.gson.Gson;
```
Comment (sarinali) [inline src/main/java/callhub/connect/drivers/MessageWebSocketAccess.java:2]: just curious why did u pick this package for converting to json object

File: src/main/java/callhub/connect/drivers/MessageWebSocketAccess.java [hunk 2]
```diff
@@ -1,24 +1,43 @@
...
     @MessageMapping("/message-employee/{sessionId}")
     @SendTo("/topic/message-customer/{sessionId}")
     public String sendMessageEmployee(@DestinationVariable String sessionId, String message) throws Exception {
-        System.out.println(message);
-        return message + " (" + sessionId + ")";
+        HashMap<String, String> response = generateResponse(message);
+        return gson.toJson(response);
+    }
+
+    private HashMap<String, String> generateResponse(String message){
```
Comment (sarinali) [inline src/main/java/callhub/connect/drivers/MessageWebSocketAccess.java:33]: i would move this to a dataaccessobject or a use case tbh

**Review by sarinali — COMMENTED** [review]
slay minor changes

File: src/main/java/callhub/connect/drivers/MessageWebSocketAccess.java (hunk 1 above)
Comment (zjayee) [reply src/main/java/callhub/connect/drivers/MessageWebSocketAccess.java:2]: I think gson is popular way for java
(and google #1)

File: src/main/java/callhub/connect/drivers/MessageWebSocketAccess.java (hunk 2 above)
Comment (zjayee) [reply src/main/java/callhub/connect/drivers/MessageWebSocketAccess.java]: done

**Review by 3milyfz — APPROVED** [review]
lgtm!

**Pull request merged**